docs/jwt/verify/interfaces/JWTVerifyOptions.md

```diff
@@ -38,6 +38,9 @@ Expected clock tolerance
 - In seconds when number (e.g. 5)
 - Parsed as seconds when a string (e.g. "5 seconds", "10 minutes", "2 hours").
```

Comment:
Can I say, I don't understand how it can be parsed in "seconds" when the input is "10 minutes". Does it mean that human readable formats will be parsed, or that "10 minutes" will be converted to `10 seconds`...

Reply:
`10 minutes` is 600, `10 seconds` is 10